src/lib/Component.coffee

```diff
@@ -178,7 +175,7 @@ class Component extends EventEmitter
       if @bracketCounter[port] isnt 0
         bracketsClosed = false
         break
-    @autoOrdering = false if bracketsClosed
+    @autoOrdering = null if bracketsClosed
```

Comment:
`Component.processOutputQueue()` should not be called at all if `autoOrdering` is `false`, but I can add a check here to avoid any possible collisions.